### PR TITLE
RNMT-1846 - Upgrade for latest version of Pushwoosh Plugin (7.8.3)

### PR DESCRIPTION
**Pull request by vmfo22**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pushwoosh-cordova-plugin",
-  "version": "6.5.3-OS4",
+  "version": "7.8.3",
```
Comment (cjaSource) [inline package.json]: Shouldn't the version also be "something-OS"?

Comment (cjaSource) [reply package.json]: Or does the upgrade "remove" our customizations?

Comment (vmfo22) [reply package.json]: Yes, should be, we remove almost customizations made except one callback. I was thinking to define that on a separated commit to raise the plugin version.

Comment (EiyuuZack) [reply package.json]: I agree, if we have changed a single line of code we should add the -OS suffix.

File: www/PushNotification.js [hunk 2]
```diff
@@ -294,7 +304,7 @@ PushNotification.prototype.postEvent = function(event, attributes) {
 };
 
 //Function: createLocalNotification
-//[android] Schedules local notification,
+//[android, ios] Schedules local notification,
```
Comment (EiyuuZack) [inline www/PushNotification.js]: Even their code documentation was missing important details 👍

**Pull request merged**